DOCS/canonical-identity-platform-how-to-manage-external-identity-providers.md

```diff
@@ -56,13 +71,20 @@ juju deploy kratos-external-idp-integrator
 juju integrate kratos kratos-external-idp-integrator
 ```
 
+[note]
```

Comment:
Yes, it's just not used in our docs or rendered in markdown. In Discourse it will be displayed similarly to this:
![image](https://github.com/user-attachments/assets/90db1e3f-2c9a-4214-9048-edc2673e1efc)